.github/ISSUE_TEMPLATE/documentation_report.yml

```diff
@@ -0,0 +1,73 @@
+name: Documentation Report
+description: File a report for additional sections or discrepancies, missing, or comment errors in our documentation.
+title: "DOC: "
+labels: ["documentation", "needs triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for helping to better our documentation! By filling out the following form in its entirety, we will be able to better what needs to be fixed or improved on. Thank you in advance for taking the time to fill out the form!
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues, known issues in release notes, and documentation.
+          required: true
+  - type: textarea
+    id: brief-description
+    attributes:
+      label: Brief Description of the Documentation Issue or Improvement
+      description: Briefly describe the issue you found and what you expected.
+      placeholder: Tell us what you see!
+      value: "The discrepancy can be found at ___ and I think it should be ___"
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of our software are you running? [Further details may be required during triage process]
+      options:
+        - 7.0.x (DREAM3DNX beta)
```

Comment:
I think this should just be DREAM3D NX (version 7). Same for the other documents